packages/devtools_app/lib/src/network/network_screen.dart

```diff
@@ -215,19 +215,23 @@ class _NetworkScreenBodyState extends State<NetworkScreenBody>
       valueListenable: _networkController.selectedRequest,
       builder: (context, selectedRequest, _) {
         return Expanded(
-          child: Split(
-            initialFractions: const [0.5, 0.5],
-            minSizes: const [200, 200],
-            axis: Axis.horizontal,
-            children: [
-              NetworkRequestsTable(
-                networkController: _networkController,
-                requests: filteredRequests,
-                searchMatchesNotifier: _networkController.searchMatches,
-                activeSearchMatchNotifier: _networkController.activeSearchMatch,
-              ),
-              NetworkRequestInspector(selectedRequest),
-            ],
+          child: GestureDetector(
+            onTap: () => FocusScope.of(context)?.unfocus(),
```

Comment:
why is this gesture detector and call to unfocus necessary?

Reply:
Text remains selected/highlighted in the SelectableText widget even when you tap anywhere else on the screen, unless another widget requests focus (TextField, Button, SelectableText - when they're tapped).
It feels natural and intuitive for the user when the SelectableText loses focus or its highlight after tapping elsewhere (which means the user is done copying).
This behaviour is on every native platform and especially in browsers too

Reply:
I see. @goderbauer @HansMuller is this focus management something that `SelectableText` should handle in the framework. It does seem intuitive that a click away from the selectable text widget would mean that the selected text should unfocus.

Reply:
That does seem like an issue we should address in the framework. Can you file an issue for it?

Reply:
Done - filed https://github.com/flutter/flutter/issues/80455